.github/workflows/config/mlc_config.json

```diff
@@ -2,35 +2,53 @@
     "ignorePatterns": [
         {
             "pattern": "https://github.com/Your_Github_Handle/owasp-mstg"
-        },{
+        },
+        {
             "pattern": "changelog"
-        },{
+        },
+        {
             "pattern": "https://github.com/commjoen/contributors-mstg"
-        },{
+        },
+        {
             "pattern": "http://127.0.0.1:8000"
-        },{
+        },
+        {
             "pattern": "https://haveibeenpwned.com"
-        },{
+        },
+        {
             "pattern": "https://build.frida.re/"
-        },{
+        },
+        {
             "pattern": "http://apt.thebigboss.org/repofiles/cydia/"
-        },{
+        },
+        {
             "pattern": "https://build.frida.re"
-        },{
+        },
+        {
             "pattern": "http://cydia.radare.org/debs/"
```

Comment:
This one returns HTTP/1.1 302 Found, what if we add that to `aliveStatusCodes`?

Reply:
302? Not really. I'd just update it to what it is looking for. I'll update this to `https` instead of `http`.

Reply:
All this links should be in the MSTG and this one is not anymore. Instead, only "https://cydia.radare.org/pool/" is used. Could you please remove this line and check if there's still an error. We'd rather have here as less "excluded links" as possible. Same goes for "https://build.frida.re", please try to remove it, it returns 200 for me.